src/app/design/resources/index.html

```diff
@@ -17,9 +17,68 @@ <h2 class="sky-section-heading">Axure kit</h2>
 </div>
 	
 <h2 class="sky-section-heading">Fonts</h2>
-<p>Desktop fonts for Blackbaud Sans, used in the Sketch and Axure kits.</p>
-<div>
-    <a href="~/assets/design/desktop-fonts.zip">Download the Blackbaud Sans desktop fonts</a>
-</div>
+<p>For desktop use and the Sketch and Axure kits, you can <a href="~/assets/design/desktop-fonts.zip">download Blackbaud Sans</a>.</p>
+<p>
+  The Blackbaud Sans typeface is available on the Blackbaud CDN for use in web and desktop applications.
+</p>
+<p>
+  For web applications, you can use Blackbaud Sans from the Blackbaud CDN in the combinations below. Depending on your application, you may find it easier to use the font through the <a href="https://github.com/blackbaud/skyux-theme">@skyux/theme package</a>.
+</p>
+
+<table>
+  <tr>
+    <th align="left" style="width:8%">Family</th>
+    <th align="left" style="width:5%">Style</th> 
+    <th align="left" style="width:4%">Weight</th>
+    <th align="left" style="width:18%">CDN link</th>
+  </tr>
+  <tr>
+    <td>Blackbaud Sans</td>
+    <td>Normal</td> 
+    <td>400</td>
+    <td><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">blackbaud-sans.woff</a></td>
+  </tr>
+  <tr>
+    <td>Blackbaud Sans</td>
+    <td>Normal</td> 
+    <td>500</td>
+    <td><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-semi-bold.woff">blackbaud-sans-semi-bold.woff</a></td>
+  </tr>
+  <tr>
+    <td>Blackbaud Sans</td>
+    <td>Normal</td> 
+    <td>600</td>
+    <td><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-bold.woff">blackbaud-sans-bold.woff</a></td>
+  </tr>
+  <tr>
+    <td>Blackbaud Sans</td>
+    <td>Italic</td> 
+    <td>400</td>
+    <td><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-italic.woff">blackbaud-sans-italic.woff</a></td>
+  </tr>
+  <tr>
+    <td>Blackbaud Sans</td>
+    <td>Light</td> 
```

Comment:
"Light" is incorrect. It should be "Normal." I confirmed this with Bobby.

Reply:
Am I missing something? The comment is resolved, but it looks like the content is unchanged.